### ISSUE TITLE
Checkout as guest

### ISSUE THREAD

**Issue by popeating** [wp-graphql/wp-graphql-woocommerce#912]
I s there a (simple) way to have the guest user to ckeckout (getting a guest token)
considering that im handling the cart in a local store (not adding product with mutations)
also i handle the user and list of orders for logged in users ina local store

**Comment by kidunot89** [comment]
@popeating Yes, there is. Have you view the [documentation](https://woographql.com/docs/configuring-graphql-client-for-user-session)

**Comment by kidunot89** [comment]
@popeating Any movement on this?

**Comment by kidunot89** [comment]
Resolved in v1.0.0 — guest checkout has been extensively improved via #1003 (checkout auth), #954 (session handler improvements), and #971 (session corruption fix).
